### PR TITLE
Wingdings Revival^2

### PR DESCRIPTION
**Pull request by fludd12**
Revives the wingdings revival PR.

What even.

:cl: Fluff12
rscadd: Greys can now choose to speak in wingdings or not. The Voice option is right below Species in character creation if Grey is selected - make sure you have it set to what you want!
/:cl:

### CODE REVIEW

**Comment by fludd12** [comment]
FRICK

File: code/modules/client/preference/preferences.dm [hunk 1]
```diff
@@ -2035,6 +2037,12 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 
 	character.add_language(language)
 
+	if(species == "Grey")
+		if(speciesprefs)
+			character.speech_span = "say"
```
Comment (tigercat2000) [inline code/modules/client/preference/preferences.dm]: You should get rid of this as well now that you changed how it works.

**Comment by Crazylemon64** [comment]
> leading merge commit >

👀

**Comment by fludd12** [comment]
FIXING THAT NOW

File: code/modules/client/preference/preferences.dm (hunk 1 above)
Comment (fludd12) [inline code/modules/client/preference/preferences.dm]: WHY IS THAT STILL A THING

Comment (fludd12) [inline code/modules/client/preference/preferences.dm]: wait a second what

Comment (fludd12) [inline code/modules/client/preference/preferences.dm]: no, that's right, I think

**Comment by fludd12** [comment]
wait what did travis do

**Comment by fludd12** [comment]
NOW I RUINED MORE THINGS

**Comment by Spacemanspark** [comment]
:clap:

**Comment by fludd12** [comment]
Fixed everything, even the tracking merge.

**Comment by Xantholne** [comment]
👏

**Comment by fludd12** [comment]
why is everyone clapping

**Comment by Krausus** [comment]
> Files changed 55

oh dear

**Comment by fludd12** [comment]
what the frick what happened

**Comment by Alexshreds** [comment]
You should git sync, pull, commit, squash/merge, then PR.... every single time.

Now you're trying to merge an older version of paracode + your changes.

**Comment by fludd12** [comment]
I noticed that, fixing the things now.

**Comment by fludd12** [comment]
Frick that didn't fix anything

**Comment by fludd12** [comment]
Everything is terrible and horrible oh god

**Comment by fludd12** [comment]
ok ok it's working now technically

**Comment by Krausus** [comment]
Wingdings should be something you have to toggle on, not the default as you have it now.

Should probably reset `speciesprefs` when the species option is changed, so it doesn't unintentionally carry over between species.

Changelog is missing.

**Comment by fludd12** [comment]
ack frick you're right

**Comment by fludd12** [comment]
okay, okay, I think it's finally not terrible :relieved:

**Comment by Spacemanspark** [comment]
uh
Why does the file state that it's deleted.

**Comment by fludd12** [comment]
what

**Comment by Spacemanspark** [comment]
I'm fucking with you

**Comment by Crazylemon64** [comment]
@fludd12 The commits here are janky, use the secret monkey technique that I taught you

**Comment by fludd12** [comment]
@Crazylemon64 I think it broke things

**Comment by fludd12** [comment]
@Crazylemon64 @tigercat2000 This is gonna need a quick review again because git flipped out.

File: code/modules/client/preference/preferences.dm [hunk 2]
```diff
@@ -2035,6 +2038,10 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 
 	character.add_language(language)
 
+	if(species == "Grey")
+		if(speciesprefs)
+			character.speech_span = "wingdings"
```
Comment (Crazylemon64) [inline code/modules/client/preference/preferences.dm]: This won't apply if the character is cloned - you'll want to put this in the greys' `on_species_change` or something to set things up accordingly. Make sure to accordingly reset it when the player's species changes away from Grey, too.

Comment (fludd12) [inline code/modules/client/preference/preferences.dm]: If I moved it completely to `on_species_change`, would it still apply to first spawning, or should it copypasta over?

Comment (Krausus) [inline code/modules/client/preference/preferences.dm]: A more complicated change would be necessary - this is the only place you should ever transfer preferences to a character. You'd have to attach it to something persistent, like the mind or DNA. In this case, probably the mind.

Comment (fludd12) [inline code/modules/client/preference/preferences.dm]: Eef, alright.

Comment (Krausus) [inline code/modules/client/preference/preferences.dm]: You cooould probably move the `speech_span` entirely to the mind, make the wingdings permanent regardless of what species you become. Once you go ☝︎✋︎👌︎👌︎☜︎☼︎✋︎💧︎☟︎, you never go back.

**Comment by fludd12** [comment]
@Crazylemon64 How's it look?

**Comment by pazneria12** [comment]
I still think that Greys would be better to have unlimited range, but limited so they cant do it while handcuffed, because wingdings are awesome, but if there is no one to hear it, theres no reason to speak it.

**Comment by fludd12** [comment]
This is just for actual speaking, and it's an option.

**Comment by Fox-McCloud** [comment]
Lavaland went through, so...you mayyy have to change this, as @Markolie pointed out.

**Comment by fludd12** [comment]
It *seems* like everything's just fine, so I'll leave it be until something spooks me.

**Comment by fludd12** [comment]
Just a ping, this is ready, so long as you guys are okay with it.

**Comment by Spacemanspark** [comment]
Can we merge this now?

**Comment by Fox-McCloud** [comment]
@Crazylemon64

**Pull request merged**